src/lib/flows.ts

```diff
@@ -460,7 +460,8 @@ export const translateFlow = async (flowName: string, options: ITranslateFlowOpt
         const targetLocale = (await pullLocales()).find((locale) => locale.name === localeName);
         const flowIntents = (await CognigyClient.indexIntents({
```

Comment:
After performing some tests, it looks `indexIntents(...)` does not support pagination! It returns only the first 25 intents of flow. On flows containing >25 intents this leads to incomplete data processing.

@XavierJordaMurria do you know if pagination should been supported by `CognigyClient.indexIntents(...)` ? If not, it needs to be added here.

Reply:
hi @tgbv , I will have a look at the cognigy API when I have some time. 
thanks for your pr

Reply:
I am closing this pr since I have fixed this issue in this other pull request.
https://github.com/Cognigy/Cognigy-CLI/pull/133